azure-pipelines.yml

```diff
@@ -315,265 +315,4 @@ jobs:
       Verbosity: 'Information'
   - publish: $(Build.Repository.LocalPath)\WebJobs.Script.Performance.App
     artifact: Performance
-
-- job: RunUnitTests
```

Comment:
Did you intentionally check this in? For testing? I'm assuming this is not a change we want to merge

Reply:
This PR is just for testing in private stamp. Going to close this one and open a new

Reply:
have made this a draft PR